### PR TITLE
Fixes exploit

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Fixes https://github.com/ParadiseSS13/Paradise/issues/19928

## Why It's Good For The Game
Exploit bad

## Changelog
:cl:
fix: Fixed an exploit with cryo storage
/:cl:

### CODE REVIEW

File: code/game/machinery/cryopod.dm [hunk 1]
```diff
@@ -110,7 +110,7 @@
 				return
 
 			var/obj/item/item = locateUID(params["item"])
-			if(!item)
+			if(!item || item.loc != src)
```
Comment (S34NW) [inline code/game/machinery/cryopod.dm:113]:
```suggestion
			if(!item || !(item in frozen_items))
```
Two ways of doing this, unsure which is best

**Comment by S34NW** [comment]
> damn paradise is quick as hell with fixing exploits. Does src include all lists in an object? If not then this might break it.

AFAIK no object that was in a storage object will remain in said object when stored. If this is not true then it would break, my comment would work though.

**Review by farie82 — APPROVED** [review]

**Comment by farie82** [comment]
@CocaColaTastesGood quick request for future reports. Can you PM me or another headcoder about these instead of making an issue report? Thanks for the report btw it is appreciated

**Comment by S34NW** [comment]
> > @CocaColaTastesGood quick request for future reports. Can you PM me or another headcoder about these instead of making an issue report? Thanks for the report btw it is appreciated
> 
> Sure, I would've posted on the forums but the bug report section is locked behind verified accounts.

We can manually verify you, are you on our discord? If so drop me a PM at `S34N#2501`

**Pull request merged**